crates/forge/src/test_execution_syscall_handler.rs

```diff
@@ -69,7 +69,7 @@ pub struct TestExecutionSyscallHandler<'a> {
 }
 
 impl<'a> TestExecutionSyscallHandler<'a> {
-    pub fn new(
+    pub fn wrap(
```

Comment:
I think `new` was fine

Reply:
It does not really tell us what is done here, since effectively we're wrapping the syscall handler with some handling logic.
It's useful for the reader imho

Reply:
Agreed with @Arcticae wrap is much better